sounds/models.py

```diff
@@ -729,8 +730,11 @@ def post_delete_comment(self, commit=True):
             self.save()
 
     def compute_crc(self, commit=True):
-        p = subprocess.Popen(["crc32", self.locations('path')], stdout=subprocess.PIPE)
-        self.crc = p.communicate()[0].split(" ")[0][:-1]
+        with open(self.locations('path'), "rb") as fp:
+            data = fp.read()
+        crc = zlib.crc32(data)
```

Comment:
this is not how you're supposed to use crc32 though...

the idea is:

```
...
with open(self.locations("path"), "rb") as f:
    for chunk in iter(lambda: f.read(4096), b''):
        crc = zlib.crc32(chunk, crc)
self.crc = '{:0>8x}'.format(crc & 0xffffffff)
...
```

Reply:
Otherwise you area loading the full file into memory...!

Reply:
Nice catch, can you update this @philtgun ?